.github/workflows/tox.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
```

Comment:
is `cancerDHC/ccdhmodel` dependent on 3.9 now (from the type assertions?), or are we saying that `cancerDHC/ccdhmodel` is somewhat dependent on `cancerDHC/sheet2linkml`, which is dependent on 3.9?